modules/modules.nix

```
@@ -152,7 +152,8 @@ let
     (loadModule ./services/unison.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/window-managers/awesome.nix { })
     (loadModule ./services/window-managers/bspwm/default.nix { condition = hostPlatform.isLinux; })
-    (loadModule ./services/window-managers/i3.nix { })
+    (loadModule ./services/window-managers/i3-sway/i3.nix { })
+    (loadModule ./services/window-managers/i3-sway/sway.nix { })
```

Could do
```suggestion
    (loadModule ./services/window-managers/i3-sway/sway.nix { condition = hostPlatform.isLinux; })
```
to restrict the module to Linux. Unfortunately the i3 module needs to stay as is for backwards compatibility reasons.